lib/publiccloud/instance.pm

```diff
@@ -919,9 +919,18 @@ sub do_systemd_analyze_time {
 
 sub upload_supportconfig_log {
     my ($self, %args) = @_;
-    $self->ssh_script_run(cmd => 'sudo supportconfig -R /var/tmp -B supportconfig -x AUDIT', timeout => 600);
-    $self->ssh_script_run(cmd => 'sudo chmod 0644 /var/tmp/scc_supportconfig.txz');
-    $self->upload_log('/var/tmp/scc_supportconfig.txz', failok => 1, timeout => 600);
+    return unless (get_var('SUPPORTCONFIG_LOG', 1));
+    # to exclude nothings, set "-"
+    my $exclude = get_var('SUPPORTCONFIG_EXCLUDE', 'AUDIT');
+    $exclude = ($exclude =~ s/[-\s]+//gr) ? '-x ' . $exclude : '';
+    my $log = "/var/tmp/scc_supportconfig";
+    my $cmd = "timeout --preserve-status 900 sudo supportconfig -R " . dirname($log) . " -B supportconfig $exclude";
+    record_info('supportconfig start', "start time: " . localtime());
+    my $err = $self->ssh_script_run(cmd => "$cmd", timeout => 960, proceed_on_failure => 1);
```

Comment:
And then reuse the variable here:

```suggestion
    my $err = $self->ssh_script_run(cmd => "$cmd", timeout => $timeout + 60, proceed_on_failure => 1);
```

Reply:
same as before told https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/23309#discussion_r2358527249: I already did it so ... 😄 
I will restore my origina version, then.